tools/CrossStackProfiler/CspFileReader.py

```diff
@@ -100,7 +100,7 @@ def printArgs(self):
         self._logger.info(self._minTimeStamp)
 
     def _checkArgsKey(self, key, type):
-        if not self._args.has_key(key):
+        if not key in self._args:
```

Comment:
```suggestion
        if key not in self._args:
```

`not in`、`is not` 是 Python 中更为推荐的用法，见 [PEP 8 other-recommendations](https://peps.python.org/pep-0008/#other-recommendations)